streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountProcessorDemo.java

```diff
@@ -54,47 +52,46 @@
  * {@code bin/kafka-console-producer.sh}). Otherwise you won't see any data arriving in the output topic.
  */
 public final class WordCountProcessorDemo {
-
-    static class MyProcessorSupplier implements ProcessorSupplier<String, String, String, String> {
+    static class WordCountProcessor implements Processor<String, String, String, String> {
```

Comment:
This might be out of scope here, but I visited this class as a reference for my docs update. It seems more natural to explicitly define the processor and then use a lambda for the supplier than to explicitly define the supplier and use a lambda for the processor.